addons/im_livechat/static/tests/embed/autopopup_tests.js

```diff
@@ -12,10 +12,13 @@ QUnit.module("autopopup");
 QUnit.test("persisted session", async () => {
     const pyEnv = await startServer();
     const livechatChannelId = await loadDefaultConfig();
+    const guestId = pyEnv["mail.guest"].create({ name: "Visitor 11" });
+    pyEnv.cookie.set("dgid", guestId);
+
```

Comment:
Unnecessary empty line, this annoys reading test at glance, because these empty newline gives wrong impression it separates 2 tests when there's only 1 test.